README.md

```diff
@@ -10,7 +10,14 @@
 
 ### Run as a Spring Boot application
 
-`mvn clean install spring-boot:run`
+Using local profile:
+`mvn clean install spring-boot:run -Dspring.profiles.active=local`
```

Comment:
Habits die hard :) I seen you do not like blank lines :)

Reply:
i'll fix this

Reply:
Still no blank line in between.

Do need to mention `clean install` by the way? I'd expect Maven to compile before service is run.